Tests/designspaceLib/designspace_v5_test.py

```diff
@@ -0,0 +1,431 @@
+                AxisLabel(userValue=800, name="XBold"),
+                AxisLabel(userValue=900, name="Black"),
+            ],
+        ),
+        Axis(
+            tag="wdth",
+            name="Width",
+            minimum=75,
+            default=100,
+            maximum=125,
+            axisOrdering=0,
+            axisLabels=[
+                AxisLabel(name="Cd", userValue=75),
+                AxisLabel(name="Normal", elidable=True, userValue=100),
+                AxisLabel(name="Ex", userValue=125),
+            ],
+        ),
+        Axis(
+            tag="ital",
+            name="Italic",
+            minimum=0,
+            default=0,
+            maximum=1,
+            axisOrdering=2,
+            axisLabels=[
+                AxisLabel(
+                    name="Upright", userValue=0, elidable=True, linkedUserValue=1
+                ),
+                AxisLabel(name="Italic", userValue=1),
+            ],
+        ),
+    ]
+
+    assert [v.asdict() for v in doc.axes] == [v.asdict() for v in expected_axes]
+
+    expected_variable_fonts = [
+        VariableFont(
+            name="AktivGroteskVF_WghtWdthItal",
+            axisSubsets=[
+                RangeAxisSubsetDescriptor(name="Weight"),
+                RangeAxisSubsetDescriptor(name="Width"),
+                RangeAxisSubsetDescriptor(name="Italic"),
+            ],
+        ),
+        VariableFont(
+            name="AktivGroteskVF_WghtWdth",
+            axisSubsets=[
+                RangeAxisSubsetDescriptor(name="Weight"),
+                RangeAxisSubsetDescriptor(name="Width"),
+            ],
+        ),
+        VariableFont(
+            name="AktivGroteskVF_Wght",
+            axisSubsets=[
+                RangeAxisSubsetDescriptor(name="Weight"),
+            ],
+        ),
+        VariableFont(
+            name="AktivGroteskVF_Italics_WghtWdth",
+            axisSubsets=[
+                RangeAxisSubsetDescriptor(name="Weight"),
+                RangeAxisSubsetDescriptor(name="Width"),
+                ValueAxisSubsetDescriptor(name="Italic", userValue=1),
+            ],
+        ),
+        VariableFont(
+            name="AktivGroteskVF_Italics_Wght",
+            axisSubsets=[
+                RangeAxisSubsetDescriptor(name="Weight"),
+                ValueAxisSubsetDescriptor(name="Italic", userValue=1),
+            ],
+        ),
+    ]
+    assert [v.asdict() for v in doc.variableFonts] == [
+        v.asdict() for v in expected_variable_fonts
+    ]
+
+
+def test_convert_v5_document_aktiv_to_v4(datadir):
+    doc = DesignSpaceDocument.fromfile(datadir / "test_v5_aktiv.designspace")
+
+    import fontTools.designspaceLib.convert5to4
+
+    variable_fonts, stylespace = fontTools.designspaceLib.convert5to4.convert5to4(doc)
+    for vf_name, vf in variable_fonts.items():
+        vf.lib["org.statmake.stylespace"] = stylespace.to_dict()
+        vf.write(datadir / "out" / (vf_name + ".designspace"))
```

Comment:
what is these convert_v5..to_v4 tests actually checking? I don't see any asserts

Reply:
I'm going to move this converter code out of fontTools for now, and bring it back later using fontTools's STAT table tools instead of statmake. I'll take into account your feedback on this code then.

Reply:
The tests are back and they're real now